# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/efivar/hotfix.patch.ia64
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- efivar-38/src/include/gcc.specs.vanilla	2024-04-19 18:57:21.054428964 +0200
+++ efivar-38/src/include/gcc.specs	2024-04-19 18:57:34.428429521 +0200
@@ -2,7 +2,7 @@
 + %{!r:%{!fpie:%{!fPIE:%{!fpic:%{!fPIC:%{!fno-pic:-fPIE}}}}}} -grecord-gcc-switches
 
 *self_spec:
-+ %{!shared:%{!static:%{!r:-pie}}} %{static:-Wl,-no-fatal-warnings -Wl,-static -static -Wl,-z,relro,-z,now} -grecord-gcc-switches
++ %{!shared:%{!static:%{!r:-pie}}} %{static:-Wl,-no-fatal-warnings -Wl,-static -static -Wl,-z,now} -grecord-gcc-switches
 
 *link:
-+ %{!static:--fatal-warnings} --no-undefined-version --no-allow-shlib-undefined --add-needed -z now --build-id %{!static:%{!shared:-pie}} %{shared:-z relro} %{static:%<pie}
++ %{!static:--fatal-warnings} --no-undefined-version --no-allow-shlib-undefined --add-needed -z now --build-id %{!static:%{!shared:-pie}} %{static:%<pie}
--- efivar-38/src/include/defaults.mk.vanilla	2024-04-19 18:58:01.967430666 +0200
+++ efivar-38/src/include/defaults.mk	2024-04-19 18:58:06.901430871 +0200
@@ -57,7 +57,6 @@
 		   -Wl,--no-undefined-version \
 		   -Wl,-z,now \
 		   -Wl,-z,muldefs \
-		   -Wl,-z,relro \
 		   -Wl,--fatal-warnings \
 		   $(call family,LDFLAGS) $(call family,CCLDFLAGS) \
 		   $(call pkg-config-ccldflags)
